src/utils/validation-rules.js

```diff
@@ -18,3 +18,23 @@ export function useTextAreaFieldRules() {
     (v) => (v && v.length <= 100) || "Input must be less than 100 characters",
   ];
 }
+
+export function useEmailFieldRules() {
+  return [
+    (v) => !!v || "Email field is required",
+    (v) => /.+@.+\..+/.test(v) || "Email must be valid",
+    (v) => (v && v.length <= 100) || "Email must be less than 100 characters",
+  ];
+}
+
+export function usePasswordFieldRules() {
+  return [
+    (v) => !!v || "Password field is required",
+    (v) =>
+      /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@#$%^&+=!])[A-Za-z\d@#$%^&+=!]{6,}$/.test(
+        v
+      ) || "Password must meet complexity requirements",
+    (v) => v.length >= 6 || "Password must be at least 6 characters long",
```

Comment:
These requirements would be needed if we wanted to create a new password. In Login form the user would be inputting an already existing password, so we don't need to do the complexity validation here.

Reply:
fixed